packages/graphback-core/package.json

```diff
@@ -47,6 +46,7 @@
   "dependencies": {
     "@graphql-tools/merge": "6.2.3",
     "@graphql-tools/utils": "6.2.3",
+    "@types/bson": "4.0.2",
```

Comment:
Hmm is this needed?

Reply:
See https://github.com/aerogear/graphback/issues/2103

Reply:
Really surprising that this has to be a "prod" dependency. I wonder if the error in #2103 occurs if the template is outside Graphback repo. If not, we probably should let this as devDependencies.

Reply:
I only experienced this when outside the repo - I am doing all testing of templates in isolation.

Reply:
I have seen this logged by some community member on graphql-cli as well before. Good change!

Reply:
Basically this should be reexported in d.ts files generated.

Reply:
Apparently it is not always a given that `@types` should be installed as a devDependency, it is not very clear to me, but it can happen. See https://github.com/microsoft/types-publisher/issues/81#issuecomment-234051338

Reply:
This happens when type is being used internally but it is reexpored indirectly. All those db types etc. When dealing with compiled versions you do not need to deal with that - it is just that d.ts files require it on API level. Some people do reexport them when compiling (that will add them to d.ts) but @craicoverflow fix is amazing for me

Reply:
Okay. I'd not expect this to be an issue for an already compiled / transpiled dependency. This is like typescript trying to recompile graphback or something.

Reply:
More like if you trying to reference graphback in your code typescript will inspect d.ts files etc..